linux-headers-5.4.0-90-generic missing

I have this problem:

> kubectl logs ws-daemon-2nd7s -c shiftfs-module-loader
+ mkdir -p /lib/modules/5.4.0-90-generic
+ ln -s /usr/src_node/linux-headers-5.4.0-90-generic /lib/modules/5.4.0-90-generic/build
+ dkms install -m shiftfs -v 1.2 -k 5.4.0-90-generic --kernelsourcedir /usr/src_node/linux-headers-5.4.0-90-generic

Creating symlink /var/lib/dkms/shiftfs/1.2/source ->
                 /usr/src/shiftfs-1.2

DKMS: add completed.
Error! Your kernel headers for kernel 5.4.0-90-generic cannot be found.
Please install the linux-headers-5.4.0-90-generic package,
or use the --kernelsourcedir option to tell DKMS where it's located

Thanks for checking @stauder.

There appears to be another issue that I've come across with VMs created in GCP/Azure. It's in the same `shiftfs-module-loader` init container

Terminal:

```
Creating symlink /var/lib/dkms/shiftfs/1.2/source ->
                 /usr/src/shiftfs-1.2

DKMS: add completed.

Preparing kernel 5.11.0-1029-gcp for module build:
(This is not compiling a kernel, just preparing kernel symbols)
Storing current .config to be restored when complete
Running Generic preparation routine
make mrproper....(bad exit status: 2)
/usr/sbin/dkms: line 1211: .config: Read-only file system
Warning: using /usr/src_node/linux-headers-5.11.0-1029-gcp/.config
(I hope this is the correct config for this kernel)
make oldconfig...(bad exit status: 2)
make prepare-all...(bad exit status: 2)

Building module:
cleaning build area...
make -j2 KERNELRELEASE=5.11.0-1029-gcp -C /usr/src_node/linux-headers-5.11.0-1029-gcp M=/var/lib/dkms/shiftfs/1.2/build...(bad exit status: 2)
Error! Bad return status for module build on kernel: 5.11.0-1029-gcp (x86_64)
Consult /var/lib/dkms/shiftfs/1.2/build/make.log for more information.
```

Logs:
```
cat /var/lib/dkms/shiftfs/1.2/build/make.log 
DKMS make.log for shiftfs-1.2 for kernel 5.11.0-1029-gcp (x86_64)
Sun Feb 13 13:28:14 UTC 2022
make: Entering directory '/usr/src_node/linux-headers-5.11.0-1029-gcp'
  CC [M]  /var/lib/dkms/shiftfs/1.2/build/shiftfs.o
/var/lib/dkms/shiftfs/1.2/build/shiftfs.c: In function 'shiftfs_fiemap':
/var/lib/dkms/shiftfs/1.2/build/shiftfs.c:727:13: error: dereferencing pointer to incomplete type 'struct fiemap_extent_info'
  if (fieinfo->fi_flags & FIEMAP_FLAG_SYNC)
             ^~
/var/lib/dkms/shiftfs/1.2/build/shiftfs.c:727:26: error: 'FIEMAP_FLAG_SYNC' undeclared (first use in this function); did you mean 'FS_XFLAG_SYNC'?
  if (fieinfo->fi_flags & FIEMAP_FLAG_SYNC)
                          ^~~~~~~~~~~~~~~~
                          FS_XFLAG_SYNC
/var/lib/dkms/shiftfs/1.2/build/shiftfs.c:727:26: note: each undeclared identifier is reported only once for each function it appears in
/var/lib/dkms/shiftfs/1.2/build/shiftfs.c: In function 'shiftfs_btrfs_ioctl_fd_restore':
/var/lib/dkms/shiftfs/1.2/build/shiftfs.c:1422:2: error: implicit declaration of function '__close_fd'; did you mean 'close_fd'? [-Werror=implicit-function-declaration]
  __close_fd(current->files, fd);
  ^~~~~~~~~~
  close_fd
cc1: some warnings being treated as errors
scripts/Makefile.build:288: recipe for target '/var/lib/dkms/shiftfs/1.2/build/shiftfs.o' failed
make[1]: *** [/var/lib/dkms/shiftfs/1.2/build/shiftfs.o] Error 1
Makefile:1849: recipe for target '/var/lib/dkms/shiftfs/1.2/build' failed
make: *** [/var/lib/dkms/shiftfs/1.2/build] Error 2
make: Leaving directory '/usr/src_node/linux-headers-5.11.0-1029-gcp'
```

--- a/.env.example
+++ b/.env.example
@@ -7,8 +7,8 @@ USER=""
 # Base domain
 DOMAIN=gitpod.domain.com
 
-# Installer version - get from werft.gitpod-dev.com
-INSTALLER_VERSION=release-2022.01.11
+# Installer version - get from https://github.com/gitpod-io/gitpod/releases
+INSTALLER_VERSION=release-2022.01.10
 
 # The email address for cert-manager wildcard SSL certificate
 LETSENCRYPT_EMAIL=my@email

--- a/setup.sh
+++ b/setup.sh
@@ -85,6 +85,12 @@ function install() {
         --server-user "${USER}" \
         --user "${USER}"
     fi
+
+    echo "Install linux-headers"
+    ssh-keyscan "${IP}" >> ~/.ssh/known_hosts
+    ssh "${USER}@${IP}" "sudo apt-get update"
+    # shellcheck disable=SC2029
+    ssh "${USER}@${IP}" "sudo apt-get install -y linux-headers-$(uname -r) linux-headers-generic"
   done
 
   kubectl get nodes -o wide
@@ -114,7 +120,6 @@ function install() {
   yq e -i ".domain = \"${DOMAIN}\"" "${CONFIG_FILE}"
   yq e -i '.workspace.runtime.containerdRuntimeDir = "/run/k3s/containerd/io.containerd.runtime.v2.task/k8s.io"' "${CONFIG_FILE}"
   yq e -i '.workspace.runtime.containerdSocket = "/run/k3s/containerd/containerd.sock"' "${CONFIG_FILE}"
-  yq e -i '.workspace.runtime.fsShiftMethod = "fuse"' "${CONFIG_FILE}"
 
   installer render --config="${CONFIG_FILE}" > gitpod.yaml
 

